sequencer.c

```diff
@@ -1033,6 +1034,24 @@ static int run_git_commit(const char *defmsg,
 		strvec_push(&cmd.args, "--cleanup=strip");
 	if ((flags & VERBATIM_MSG))
 		strvec_push(&cmd.args, "--cleanup=verbatim");
+	if ((flags & EDIT_MSG) && (flags & INLINE_MSG)) {
+		int ret, size = opts->content_len + opts->subject_len + 1;
+		char *message;
+
+		if (size <= 1)
+			return error(_("no valid message length"));
+
+		message = malloc(size);
```

Comment:
Please use `xmalloc()`; This will handle out-of-memory scenarios by exiting instead of crashing in the next line.